src/main/scala/io/iohk/ethereum/keystore/KeyStore.scala

```diff
@@ -66,8 +66,8 @@ class KeyStoreImpl(keyStoreDir: String, secureRandom: SecureRandom) extends KeyS
 
   private def init(): Unit = {
     val dir = new File(keyStoreDir)
-    val res = Try(dir.mkdirs()).filter(identity)
-    res.failed.foreach(ex => log.error(s"Could not initialise keystore directory ($dir): $ex"))
+    val res = Try(dir.exists() || dir.mkdirs()).filter(identity)
```

Comment:
I forgot to mention you might need to check it's a folder and not a file (dir.isDirectory) This should encompass the `exists`.  Sorry for wrong steer. Also `isDirectory` has no side effects so no `()`.